view/theme/frio/templates/search_item.tpl

```diff
@@ -143,20 +143,19 @@
 
 			<p class="wall-item-actions">
 				{{* Action buttons to interact with the item (like: like, dislike, share and so on *}}
-				<span class="wall-item-actions-left">
 					{{if $item.threaded}}{{/if}}
 
 					{{* Buttons for like and dislike *}}
 					{{if $item.vote}}
 						{{if $item.vote.like}}
-					<button type="button" class="btn btn-defaultbutton-likes{{if $item.responses.like.self}} active" aria-pressed="true{{/if}}" id="like-{{$item.id}}" title="{{$item.vote.like.0}}" onclick="doActivityItemAction({{$item.id}}, 'like'{{if $item.responses.like.self}}, true{{/if}});">{{$item.vote.like.0}}</button>
+					<button type="button" class="btn btn-defaultbutton-likes{{if $item.responses.like.self}} active" aria-pressed="true{{/if}}" id="like-{{$item.id}}" title="{{$item.vote.like.0}}" onclick="doActivityItemAction({{$item.id}}, 'like'{{if $item.responses.like.self}}, true{{/if}});"></button>
```

Comment:
Why is this `<button>` tag empty? Where's the icon display defined? It should be inside this tag.

Reply:
it was empty before.
I just looked für button-like and changed the Button-text.
Probably its not necessary in the search-item.tpl ?

Reply:
Were you able to witness a visual difference after editing this template file? If yes, in what context? I can't seem to be able to show these buttons in the first place in the search results where I know this template is used.

Reply:
I could not see any visual difference.
I only changed the search_item.pl because it corresponded exactly to the schema of the wall_thread.tpl.
The only call I could find was in the conversation.php and there only as temp-definition.
Maybe changing the search-item.tpl is not necessary. Unfortunately, I don't know why there was a button text there :(

Reply:
It's okay, I believe there was an attempt to allow interactions with individual posts in the search view that was later abandoned for some reason, so now the button don't show by default even though there are provisions for them in the template.